metrics/web/test/features/auth/domain/usercases/test_utils/user_repository_stub.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:metrics/features/auth/domain/entities/user.dart';
+import 'package:metrics/features/auth/domain/repositories/user_repository.dart';
+import 'package:rxdart/rxdart.dart';
+
+class UserRepositoryStub implements UserRepository {
+  final BehaviorSubject<User> _userUpdatesStream = BehaviorSubject();
```

Comment:
We should decide whether we want to document those test classes - as @vladlaba  usually documents and you not.